pkgs/servers/http/mini-httpd/default.nix

```diff
@@ -8,6 +8,8 @@ stdenv.mkDerivation rec {
     sha256 = "1x4b6x40ymbaamqqq9p97lc0mnah4q7bza04fjs35c8agpm19zir";
   };
 
+  configureFlagsArray = [ "CFLAGS=-ansi" ];
```

Comment:
Lists in Nix are concatened to a space-separated string before being passed to the shell, thus it's impossible to pass `configureFlagsArray` this way; that variable must be assigned in shell code, i.e. in the `preConfigure` hook. In this particular case, that snipped works only because the list contains just a single item,